crates/modelardb_compression/src/lib.rs

```diff
@@ -45,7 +45,7 @@ pub const GORILLA_MAXIMUM_LENGTH: usize = 50;
 /// Compress `uncompressed_timestamps` using a start time, end time, and a
 /// sampling interval if regular and delta-of-deltas followed by a variable
 /// length binary encoding if irregular. `uncompressed_values` is compressed
-/// within `error_bound` using the model types in [`models`](models).
+/// within `error_bound` using the model types in [`models`].
```

Comment:
As the lines in the middle of the doc comment has been shortened, the doc comment should be reformatted so each line fits as many words as possible without exceeding the maximum line width. Editors often has functionality to automate this task, e.g., Emacs provides the `fill-paragraph` command while Neovim and Vim provides similar functionality through the `gq` keybinding.

Reply:
Fixed.

Reply:
This seems to be fixed so I have marked it as resolved.